src/ol/attribution.js

```diff
@@ -53,23 +55,50 @@ ol.Attribution.prototype.getHTML = function() {
 
 
 /**
+ * When `opt_tileGrid` and `opt_projection` are provided, tile ranges from
+ * tiles that are wrapped around the x-axis will also be matched.
  * @param {Object.<string, ol.TileRange>} tileRanges Tile ranges.
+ * @param {ol.tilegrid.TileGrid=} opt_tileGrid Tile grid.
+ * @param {ol.proj.Projection=} opt_projection Projection.
```

Comment:
Any reason for making `tileGrid` and `projection` optional arguments? As far as I can tell, `intersectsAnyTileRange` is always passed a tile grid and a projection.